concordia/language_model/local_models.py

```diff
@@ -0,0 +1,307 @@
+from concordia.language_model import language_model
+from concordia.utils import measurements as measurements_lib
+
+from langchain.llms import Ollama
+from langchain.callbacks.manager import CallbackManager
+from langchain.callbacks.streaming_stdout import StreamingStdOutCallbackHandler
+
+from collections.abc import Collection, Sequence
+from string import Template
+from typing_extensions import override
+
+import logging
+import re
+import torch
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+_MAX_MULTIPLE_CHOICE_ATTEMPTS = 5
```

Comment:
We have language_models/retry_wrapper.py to handle retry logic. Is it possible to omit all the retry logic from this class and use that?

Reply:
done